vscode/microsoft-kiota/src/utilities/progress.ts

```diff
@@ -8,9 +8,13 @@ function openTreeViewWithProgress<T>(callback: () => Promise<T>): Thenable<T> {
     cancellable: false,
     title: vscode.l10n.t("Loading...")
   }, async (progress, _) => {
-    const result = await callback();
-    await vscode.commands.executeCommand(treeViewFocusCommand);
-    return result;
+    try {
+      const result = await callback();
+      await vscode.commands.executeCommand(treeViewFocusCommand);
+      return result;
+    } catch (error) {
```

Comment:
remove the try catch as it does not provide any value and messes up the callstack

Reply:
Without it the error is swallowed and when generation doesn't work, the error doesn't come up

Reply:
my suggestion is to remove the entire try catch, not just the throw

Reply:
confirmed that wrapping it elsewhere works